### PR TITLE
Entity mixin

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: plugins/module_utils/foreman_helper.py [hunk 1]
```diff
@@ -499,10 +509,14 @@ def find_operatingsystems(self, names, **kwargs):
...
 
     def find_puppetclass(self, name, environment=None, params=None, failsafe=False, thin=None):
+        if thin is None:
+            thin = self._thin_default
         if environment:
             scope = {'environment_id': environment}
         else:
-            scope = None
+            scope = {}
+        if params is not None:
```
Comment (evgeni) [inline plugins/module_utils/foreman_helper.py:518]: funny, this was never used before? :)

Comment (mdellweg) [reply plugins/module_utils/foreman_helper.py:518]: No, it wasn't. Also noone ever specified `thin` in the `entity_opts`. But you _can_ do that now...

File: plugins/modules/content_view_filter.py [hunk 2]
```diff
@@ -168,7 +168,7 @@
 
 RETURN = ''' # '''
 
-from ansible_collections.theforeman.foreman.plugins.module_utils.foreman_helper import KatelloAnsibleModule, _foreman_spec_helper
+from ansible_collections.theforeman.foreman.plugins.module_utils.foreman_helper import KatelloMixin, ForemanStatelessEntityAnsibleModule
```
Comment (evgeni) [inline plugins/modules/content_view_filter.py:171]: Should we have a `KatelloStatelessEntityAnsibleModule`?

Comment (mdellweg) [reply plugins/modules/content_view_filter.py:171]: Maybe if we use it more than once?

Comment (evgeni) [reply plugins/modules/content_view_filter.py:171]: Yeah, I somehow read the diff as if we'd do that. But the other module is `setting`, so no.

**Pull request merged**